app/javascript/App/bundles/NewsfeedPage/NewsfeedPage.tsx

```diff
@@ -33,24 +32,83 @@ interface Props extends RouteComponentProps<any> {
   isNewsfeedReady: boolean,
   isCoinlistLoading: boolean,
   isCoinlistReady: boolean,
-  fetchNewsItemsForCoin: (coinSlug: string) => void,
-  fetchAllNewsItems: () => void, 
-  fetchMoreNewsItems: () => any, 
+  fetchNewsItemsForCoin: (coinSlug: string) => Promise<NewsItem[]>,
+  fetchAllNewsItems: () => Promise<NewsItem[]>, 
+  fetchMoreNewsItems: () => Promise<NewsItem[]>,
+  fetchNewNewsItems: () => Promise<NewsItem[]>,
 };
 
 interface State {
   initialRenderTips: boolean,
   newsfeedTips: boolean,
+  unseenNews: number[],
+  isWindowFocused: boolean,
 };
 
 class NewsfeedPage extends React.Component<Props, State> {
+
+  private documentTitle = document.title;
+
   state = {
     initialRenderTips: false,
     newsfeedTips: true,
+    unseenNews: [],
+    isWindowFocused: true,
+  }
+
+  startPollingNews = () => {
+    setTimeout(() => {
+      this.fetchNewNewsItems().then(() => {
+        this.startPollingNews();
+      })
+    }, POLLING_TIMEOUT)
+  }
+
+  updateTitle = (news?: NewsItem[]) => {
+    if(typeof news !== 'undefined') {
+      if (news.length === 0) return;
+      document.title = news.length + ' | ' + this.documentTitle;
+      return;
+    }
+
+    if (this.state.unseenNews.length === 0) {
```

Comment:
this expression will return a boolean: 
`this.state.unseenNews.length`

seems unnecessary to check that it's exactly equal to zero

Reply:
Rewrote to short expression